tox.ini

```diff
@@ -16,7 +16,7 @@ basepython =
      py37: {env:TOXPYTHON:python3.7}
 passenv = NUMBER_OF_PROCESSORS
 deps =
-     pytest
+     pytest==3.4.2
```

Comment:
`pytest3.4.2` does not support `python3.3`...

Reply:
I will remove the Python 3.3 entry in travis :)